docs/content/en/docs/clustermgmt/security/manually-renew-certs.md

```diff
@@ -23,20 +26,20 @@ Certificates in external etcd and control plane nodes expire after 1 year. The f
 |                       | apiserver                |
 |                       | front-proxy-client       |
 
-You can renew all certificates in the table except `ca` by following the steps given below. Note that the commands used in Bottlerocket nodes are more sophisticated.
+You can rotate certificates by following the steps given below. You cannot rotate the `ca` certificate because it is the root certificate. Note that the commands used for Bottlerocket nodes are different than those for Ubuntu and RHEL nodes.
 
-### External etcd node
+#### External etcd nodes
 
-If the cluster has external etcd, you need to renew the certificates in etcd nodes first.
+If your cluster is using external etcd nodes, you need to renew the etcd node certificates first. 
 
 {{% alert title="Note" color="primary" %}}
-You can find out whether any external etcd node exists by running the following command:
-```
+You can check for external etcd nodes by running the following command:
+```bash
 kubectl get etcdadmcluster -A
 ```
 {{% /alert %}}
 
-1. ssh into the etcd node and run the following commands:
+1. SSH into each etcd node and run the following commands. Etcd automatically detects the new certificates after the `etcdadm join` command completes.
```

Comment:
```suggestion
1. SSH into each etcd node and run the following commands. Etcd automatically detects the new certificates and deprecates its old certificates.
```
since Etcd is detecting new certificates before `etcdadm join` command